src/twisted/logger/_format.py

```diff
@@ -164,6 +166,51 @@ def formatEventAsClassicLogText(
     return eventText + "\n"
 
 
+def keycall(key: str, get: Callable[[str], Any]) -> PotentialCallWrapper:
```

Comment:
Not sure if this is a better name

```suggestion
def keyGetter(key: str, getter: Callable[[str], Any]) -> PotentialCallWrapper:
```

Reply:
It doesn't get the result though, it calls the result :)